vector-config/src/main/res/values/config-settings.xml

```diff
@@ -38,7 +38,7 @@
 
     <!-- Level 1: Labs -->
     <bool name="settings_labs_thread_messages_default">false</bool>
-    <bool name="settings_labs_new_app_layout_default">false</bool>
+    <bool name="settings_labs_new_app_layout_default">true</bool>
```

Comment:
It won't work only for people who changed value manually, means enabled AppLayout and then disabled it. For those who didn't change from default value it will work

Reply:
Cool, I was not sure. Then this is really fine!